.gitignore

```diff
@@ -12,3 +12,4 @@ node_modules
 .idea
 env
 frontend/archive.tar.gz
+image-sync/oc-mirror/generated.yml
```

Comment:
is this required after you switched to directly go for kubectl apply?

Reply:
It certainly helps with local debugging